### DISCUSSION TITLE
accelerate nnunet inference with TensorRT and Lazy resampling

### DISCUSSION THREAD

**Discussion by JunMa11** [Project-MONAI/tutorials#1421] [Ideas]
Dear MONAI team,

Thanks for the awesome work. The new features in the latest release is really amazing!

The inference speed of nnunet could be a bottleneck under low-resource settings even if close the TTA. The two new features in 1.2 (TensorRT and lazy resampling) have great potential to speed up the nnunet inference process. 
I'm wondering if there is any plan in this direction.

Thanks again for your tenacious efforts to make monai more and more powerful and user-friendly!

**Comment by wyli** [comment]
Many thanks @JunMa11. Are you referring to this integration of nnUNet? https://github.com/Project-MONAI/tutorials/tree/main/nnunet we don't have a plan to further optimize this, but we are prioritising features for upcoming 1.3 and beyond. Please let us know any potential ideas..
